lib/Client.php

```diff
@@ -239,7 +239,7 @@ public function poll(): bool
 
                     $curlResult['request'] = $request;
 
-                    if ($errorCallback) {
+                    if (is_callable($errorCallback)) {
```

Comment:
Could we instead declar wthe type of the propery?

Reply:
`phpstan-strict-rules` wants all `if` checks to be real booleans.
We can only call `$errorCallback` if it is a callback anyway.

`sendAsyncInternal` already has this passed in as a `callback`.

Reply:
Declaring the type as `callable` will not stop the strict rule complaining that the `if` condition is not a real boolean.

Reply:
I added details of the "array shape" of `curlMultiMap` (see below in the diffs).